src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -561,6 +562,7 @@ public virtual PSCredential Credential
         /// to override the policy setting
         /// </remarks>
         [Parameter(ParameterSetName = PSRemotingBaseCmdlet.ComputerNameParameterSet)]
+        [Parameter(ParameterSetName = PSRemotingBaseCmdlet.SSHHostParameterSet)]
```

Comment:
It looks like we are Ok.  I assume that this works and doesn't result in a "cannot resolve parameter set" error?
```powershell
New-PSSession -HostName localhost -Port 22
```
Also please check this again when you implement Invoke-Command SSH parameter sets
``` powershell
Invoke-Command -HostName localhost -Port 22 -ScriptBlock { "Hello" }
Invoke-Command localhost { "Hello" } -Port 5895
```
I think this will also be Ok but it does have more complex parameter sets.